frontend/app/src/pages/defi/airdrops/index.vue

```diff
@@ -45,9 +46,12 @@ const rows = computed<(Airdrop & { index: number })[]>(() => {
   const data = filterByAddress(get(airdrops), addresses);
   return data.filter((airdrop) => {
     const currentStatus = get(status);
+    const currentTime = Date.now() / 1000;
```

Comment:
just small question, is the cutoff time saved in UTC
If yes, then I think you need to use utc time instead here

Reply:
Hi, I believe Date.now() is UTC.  source: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/UTC

Reply:
oh yeah, you're right, cool then